web/impact/impact/tests/api_test_case.py

```diff
@@ -129,7 +132,30 @@ def assert_notified(self,
                 self.assertTrue(any([
                     message in email.body for email in emails]))
         if subject:
-            self.assertIn(subject, [email.subject for email in emails])
+            self.assertIn(subject, [email for email in emails])
+
+    def assert_email_attachments(self,
```

Comment:
This seems to be specifically about one particular email attachment, namely the ical calendar attachment. I would prefer not to define a special-purpose assertion in this general test case.

Reply:
@geofrocker this still needs to be addressed.

Reply:
addressed